.gitignore

```diff
@@ -11,7 +11,6 @@
 *.swp
 
 # Generated files and folders
-protoc-gen-grpc-cpp
```

Comment:
Was this auto-generated or any context on why it's removed?

Reply:
Sorry - it's a remnant (dead code).